src/snowflake/connector/auth/keypair.py

```diff
@@ -148,26 +162,24 @@ def calculate_public_key_fingerprint(private_key):
 
         return public_key_fp
 
-    def update_body(self, body):
+    def update_body(self, body: dict[Any, Any]) -> None:
         body["data"]["AUTHENTICATOR"] = KEY_PAIR_AUTHENTICATOR
         body["data"]["TOKEN"] = self._jwt_token
 
-    def assertion_content(self):
+    def assertion_content(self) -> str:
         return self._jwt_token
 
     def should_retry(self, count: int) -> bool:
         return count < self._jwt_retry_attempts
 
-    def get_timeout(self) -> int:
-        return self._jwt_cnxn_wait_time.seconds
-
     def handle_timeout(
         self,
         authenticator: str,
         service_name: str | None,
         account: str,
         user: str,
         password: str | None,
+        **kwargs: Any,
```

Comment:
the protocol now is keyword argument only:
```suggestion
    def handle_timeout(
        self,
        *
        authenticator: str,
        service_name: str | None,
        account: str,
        user: str,
        password: str | None,
        **kwargs: Any,
```